build/configs/rtl8721csm/hello/defconfig

```diff
@@ -282,7 +283,7 @@ CONFIG_PREALLOC_TIMERS=2
 # Tasks and Scheduling
 #
 CONFIG_INIT_ENTRYPOINT=y
-CONFIG_RR_INTERVAL=100
+CONFIG_RR_INTERVAL=10
```

Comment:
Is there any specific reason to decrease the RR interval only in this config?

Reply:
Decreasing the RR interval is part of wifi performance improvement. All the wifi performance related parameters are tuned only on hello example.